src/Chapter4.hs

```diff
@@ -611,6 +645,15 @@ Implement the 'Monad' instance for our lists.
   maybe a few) to flatten lists of lists to a single list.
 -}
 
+--copypasted :(
+concatList :: List (List a) -> List a
+concatList Empty = Empty
+concatList (Cons x xs) = append x (concatList xs)
+
+instance Monad (List) where
+    (>>=) :: List a -> (a -> List b) -> List b
+    Empty >>= _ = Empty
```

Comment:
Actually, you don't need to handle this case separately. `fmap` and `concatList` return `Empty` on `Empty`, so this is already converted :lock: